retrieval_service/datastore/helpers.py

```diff
@@ -12,25 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Union
+
 import sqlparse
 
 
-def format_sql(sql: str, params):
+def format_sql(sql: str, params: Union[tuple, dict]):
```

Comment:
Why can't we just consolidate our use between tuples or dicts?

Reply:
Thanks for the suggestion. That simplifies the `format_sql`.

Reply:
Still looks the same to me?

Reply:
My bad. I should've clarified that this is already addressed in #495, which consolidates the non-cloud Postgres client. Since that was the only client using `format_sql` with `params` as a `tuple`,  we can now safely remove that functionality. Also agree with your comment to merge these 2 PRs. Thanks!